docs/tutorials/multi_contrast.ipynb

```diff
@@ -227,22 +187,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
    "id": "74e19578-03d4-4295-b6e2-a65fb2508c78",
    "metadata": {},
    "outputs": [],
    "source": [
     "head_thickness = 12\n",
     "tail_thickness = 20\n",
-    "head_solvent_surface_coverage = 0.5\n",
-    "tail_solvent_surface_coverage = 0.0\n",
+    "head_solvation = 0.5\n",
+    "tail_solvation = 0.0\n",
```

Comment:
THis is not a good example as the head and tail groups have to be equal, so the solvation should be the same between them